pkg/operator/platform/platform.go

```diff
@@ -75,64 +62,3 @@ func getClient(explicitKubeconfig string) (client.Client, error) {
 
 	return dynamicClient, nil
 }
-
-func GetFeatureGates(ctx context.Context) (featuregates.FeatureGate, error) {
```

Comment:
Wondering if, now you have plumbing for getting feature gates, whether it would be worthwhile keeping the feature gate access plumbed through for future features? 

The pattern we've been suggesting is to start the feature gate watcher in `main.go` and pass it into your controller, then, when you want to add other feature gated features later down the line, all you need do is use the featuregateaccessor, rather than have to reintroduce the plumbing for it

Reply:
It should be trivial to `git revert` this, and dead code rots quickly, so I don't really know that I would personally prefer to do what you suggest. If you feel strongly, it's easy enough to do, though. Please let me know.

Reply:
Happy for you to continue removing for now since this isn't the "recommended" way anyway, long term though I'm trying to promote everyone have the feature gate code implemented and ready to make it easier to add new features. Carry on as you were